src/core/default/mappers/simple/lambda_deployer.py

```diff
@@ -214,6 +213,7 @@ def _remove_simple_lambda(
     output_task.update(comment=f"Removed resources for lambda {cloud_id}")
 
 
+#ANIBAL return type does not match
```

Comment:
For now, the type signature can be Dict. Any mapper function that handles Updates to a resource must return a Dict that represents the new Cloud Output of the resource. The return value is then stored in the Backend as the Cloud Output for the resource. 

Eventually, it might be better to define a more explicit return type that conforms to the exact Dict scheme we expect per resource, but for now Dict is fine.